test/test_data/benchmark_https_client_test_envoy.yaml

```diff
@@ -0,0 +1,50 @@
+admin:
```

Comment:
Is this so different that it warrants a separate YAML? Also, have you considered using https://github.com/envoyproxy/envoy/blob/master/test/config/utility.h to generate these?